### PR TITLE
4954: Fix anchors bug

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/preview/ui.test.js [hunk 1]
```diff
@@ -78,7 +78,9 @@ test('should render the marketing preview page last updated date', async (t) =>
     .expect(await extractInnerText(lastUpdated)).eql('Solution information last updated: 15 March 1996');
 });
 
-test('when no existing marketing data - The solution description section should not be rendered', async (t) => {
+fixture('Show marketing preview page - no marketing data');
```
Comment (alibombaye) [inline app/pages/preview/ui.test.js]: oh yeah!!!

File: app/pages/preview/ui.test.js [hunk 2]
```diff
@@ -111,16 +181,7 @@ test('when existing marketing data - The solution description section and all qu
...
-test('when no existing marketing data - The features section should not be rendered', async (t) => {
-  await pageSetup(t);
-  const featuresSection = Selector('[data-test-id="view-features"]');
-
-  await t
-    .expect(featuresSection.exists).notOk();
-});
-
-test('when existing marketing data - The features section should be rendered and the features displayed', async (t) => {
+test('Features section should be rendered and the features displayed', async (t) => {
```
Comment (alibombaye) [inline app/pages/preview/ui.test.js]: mmm blank line pretty please

File: app/pages/preview/ui.test.js [hunk 3]
```diff
@@ -136,16 +197,7 @@ test('when existing marketing data - The features section should be rendered and
...
-test('when no existing marketing data - The capabilities section should not be rendered', async (t) => {
-  await pageSetup(t);
-  const featuresSection = Selector('[data-test-id="view-capabilities"]');
-
-  await t
-    .expect(featuresSection.exists).notOk();
-});
-
-test('when existing marketing data - The capabilities section should be rendered and the capabilities displayed', async (t) => {
+test('Capabilities section should be rendered and the capabilities displayed', async (t) => {
```
Comment (alibombaye) [inline app/pages/preview/ui.test.js]: and here

File: app/pages/preview/ui.test.js [hunk 4]
```diff
@@ -168,16 +220,7 @@ test('when existing marketing data - The capabilities section should be rendered
...
-test('when no existing marketing data - The integrations section should not be rendered', async (t) => {
-  await pageSetup(t);
-  const integrationsSection = Selector('[data-test-id="view-integrations"]');
-
-  await t
-    .expect(integrationsSection.exists).notOk();
-});
-
-test('when existing marketing data - The integrations section should be rendered', async (t) => {
+test('Integrations section should be rendered', async (t) => {
```
Comment (alibombaye) [inline app/pages/preview/ui.test.js]: and here too

File: app/pages/preview/ui.test.js [hunk 5]
```diff
@@ -190,16 +233,7 @@ test('when existing marketing data - The integrations section should be rendered
...
-test('when no existing marketing data - The implementation timescales section should not be rendered', async (t) => {
-  await pageSetup(t);
-  const integrationsSection = Selector('[data-test-id="view-implementation-timescales"]');
-
-  await t
-    .expect(integrationsSection.exists).notOk();
-});
-
-test('when existing marketing data - The implementation timescales section should be rendered', async (t) => {
+test('Implementation timescales section should be rendered', async (t) => {
```
Comment (alibombaye) [inline app/pages/preview/ui.test.js]: you really don't like them blank lines

File: app/pages/preview/ui.test.js [hunk 6]
```diff
@@ -211,17 +245,7 @@ test('when existing marketing data - The implementation timescales section shoul
...
-  await pageSetup(t);
-
-  const clientApplicationTypesSection = Selector('[data-test-id="view-client-application-types"]');
-
-  await t
-    .expect(clientApplicationTypesSection.exists).notOk();
-});
-
-test('when existing marketing data - The client application type section and browser-based section should be rendered', async (t) => {
+test('Client application type section and browser-based section should be rendered', async (t) => {
```
Comment (alibombaye) [inline app/pages/preview/ui.test.js]: Is there a reason for getting rid of these?

File: app/pages/preview/ui.test.js [hunk 7]
```diff
@@ -283,8 +307,7 @@ test('when existing marketing data - The client application type section and bro
     .expect(await extractInnerText(additionalInformationRow.find('div[data-test-id="view-section-table-row-component"]'))).eql('Some browser additional information')
     .expect(additionalInformationRow.find('div[data-test-id="view-section-table-row-vertical"]').exists).ok();
 });
-
-test('when existing marketing data - The client application type section and native-mobile section should be rendered', async (t) => {
+test('Client application type section and native-mobile section should be rendered', async (t) => {
```
Comment (alibombaye) [inline app/pages/preview/ui.test.js]: 😨

File: app/pages/preview/ui.test.js [hunk 8]
```diff
@@ -366,8 +389,7 @@ test('when existing marketing data - The client application type section and nat
     .expect(await extractInnerText(additionalInformationRow.find('div[data-test-id="view-section-table-row-component"]'))).eql('It is possible that it may install on other platforms or versions not listed in this section. However, support is limited to systems that meet the minimum requirements.')
     .expect(additionalInformationRow.find('div[data-test-id="view-section-table-row-vertical"]').exists).ok();
 });
-
-test('when existing marketing data - The client application type section and native-desktop section should be rendered', async (t) => {
+test('client application type section and native-desktop section should be rendered', async (t) => {
```
Comment (alibombaye) [inline app/pages/preview/ui.test.js]: Didn't think they would be so many

File: app/pages/preview/ui.test.js (hunk 7 above)
Comment (kat1906) [reply app/pages/preview/ui.test.js]: I didn't mean to commit this file 😄

**Pull request merged**